libraries/AP_Common/Bitmask.h

```diff
@@ -65,7 +72,16 @@ class Bitmask {
         }
         return true;
     }
-    
+
+    // return number of bits set
+    uint16_t count() {
+        uint16_t sum = 0;
+        for (uint16_t i=0; i<numwords; i++) {
+            sum += bits[i];
```

Comment:
The bitmask can safely be longer than the path, as long as the default for the bitmask is 1111...